### DISCUSSION TITLE
AWS OnChat error

### DISCUSSION THREAD

**Discussion by saiftech1** [HyperLife1119/OnChat#5] [Q&A]
Good Morning, I have make the nacessary configuration on AWS but i was getting error /home/ubuntu/OnChat/vendor/topthink/framework/src/think/console/output/driver/Console.php:203
-bash: /home/ubuntu/OnChat/vendor/topthink/framework/src/think/console/output/driver/Console.php:203: No such file or directory, Kindly help

**Comment by HyperLife1119** [comment]
In which step did you encounter this error?

**Reply by saiftech1** [reply]
While Running php think onchat init it shows  [TypeError] Argument 1 passed to think\Container::make() must be of the type string, null given, called in /home/ubuntu/OnChat/app/core/storage/Storage.php on line 26, while running php think onchat start, it return the above error, but all the requirements mention have been meet.
Thank you

**Reply by HyperLife1119** [reply]
Please show your config/storage.php file

**Reply by saiftech1** [reply]
<?php

use app\core\storage\driver\Local;
use app\core\storage\driver\Oss;

// +----------------------------------------------------------------------
// | 文件储存设置
// +----------------------------------------------------------------------

return [
    'default' => env('storage.driver', 'oss'),
    'stores'  => [
        'oss' => [
            'driver'                  => Oss::class,
            // 阿里云账户AccessKeyId
            'access_key_id'           => env('oss.access_key_id', ''),
            // 阿里云账户AccessKeySecret
            'access_key_secret'       => env('oss.access_key_secret', ''),
            // 地域节点（可填写自己绑定的域名）
            'endpoint'                => env('oss.endpoint', 'https://oss-cn-shanghai.aliyuncs.com'),
            // Bucket
            'bucket'                  => env('oss.bucket', 'onchat'),
            // 包括自定义分隔符，图片样式名：缩略图
            'img_stylename_thumbnail' => env('oss.img_stylename_thumbnail', 'thumbnail'),
        ],

        'local' => [
            'driver' => Local::class,
        ]
    ]
];

**Reply by saiftech1** [reply]
How can i modified it to AWS?

**Reply by HyperLife1119** [reply]
And show your .env file

**Reply by HyperLife1119** [reply]
Please check:
```env
[STORAGE]
DRIVER = us
```
This will be used as the key of the stores array in the config/storage.php file.

**Reply by HyperLife1119** [reply]
Also, you must also fill in the stmp related configuration

**Reply by saiftech1** [reply]
I have done all the changes but still keep getting the same error.

**Reply by HyperLife1119** [reply]
Please show your changes.

**Comment by saiftech1** [comment]


**Reply by saiftech1** [reply]
<?php

use app\core\storage\driver\Local;
use app\core\storage\driver\Oss;

// +----------------------------------------------------------------------
// | 文件储存设置
// +----------------------------------------------------------------------

return [
    'default' => env('storage.driver', 'local'),
    'stores'  => [
        'oss' => [
            'driver'                  => Oss::class,
            // 阿里云账户AccessKeyId
            'access_key_id'           => env('oss.access_key_id', ''),
            // 阿里云账户AccessKeySecret
            'access_key_secret'       => env('oss.access_key_secret', ''),
            // 地域节点（可填写自己绑定的域名）
            'endpoint'                => env('oss.endpoint', 'https://oss-cn-shanghai.aliyuncs.com'),
            // Bucket
            'bucket'                  => env('oss.bucket', 'onchat'),
            // 包括自定义分隔符，图片样式名：缩略图
            'img_stylename_thumbnail' => env('oss.img_stylename_thumbnail', 'thumbnail'),
        ],

        'local' => [
            'driver' => Local::class,
        ]
    ]
];

**Comment by saiftech1** [comment]


**Reply by HyperLife1119** [reply]
Please 
[STORAGE]
DRIVER = us 
changed to 
[STORAGE]
DRIVER = local

**Reply by saiftech1** [reply]
Okay

**Reply by HyperLife1119** [reply]
Do you still encounter the same error now?

**Reply by saiftech1** [reply]
Yes

**Comment by HyperLife1119** [comment]
![image](https://user-images.githubusercontent.com/41798664/125196248-bc27f980-e28b-11eb-9b84-3170cb34913e.png)
Check if the port has been released and try to modify the ip to 127.0.0.1
